videomamba/video_sm/models/videomamba_pretrain.py

```diff
@@ -236,7 +238,10 @@ def get_position_angle_vec(position):
     return  torch.tensor(sinusoid_table, dtype=torch.float, requires_grad=False).unsqueeze(0) 
 
 
-class VisionMamba(nn.Module):
+class VisionMamba(nn.Module, 
+                  library_name="video_mamba",
+                  repo_url="https://github.com/OpenGVLab/VideoMamba",
+                  tags=["video-classification"]):
```

Comment:
```suggestion
class VisionMamba(
    nn.Module,
    PyTorchModelHubMixin
    library_name="video_mamba",
    repo_url="https://github.com/OpenGVLab/VideoMamba",
    tags=["video-classification"]
):
```

@NielsRogge  I believe the `TypeError: VisionMamba.__init_subclass__() takes no keyword arguments` error you are getting is because you forgot to inherit from `PyTorchModelHubMixin` here.